enzyme/tests/Page.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import renderer from 'react-test-renderer';
+import Page from '../../src/components/Page/Page';
+import chai from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+chai.should();
+chai.use(sinonChai);
+
+describe('Page component', () => {
+
+    let mySandBox: sinon.SinonSandbox;
+
+    beforeEach(async () => {
+        mySandBox = sinon.createSandbox();
+    });
+
+    it('should render the expected snapshot', async () => {
+        const component: any = renderer
+            .create(<Page activeSmartContract="penguinContract@0.0.1"/>)
+            .toJSON();
+        expect(component).toMatchSnapshot();
+    });   
+
+    afterEach(async () => {
```

Comment:
Could we move `afterEach` below `beforeEach` but before `it` as we do it similarly in our extension tests.